Globals.cpp

```diff
@@ -746,12 +746,14 @@ global_settings::ConfigParse(cParser &Parser) {
         }
         else if (token == "uartfeature")
         {
-            Parser.getTokens(4);
+            Parser.getTokens(6);
             Parser
                 >> uart_conf.mainenable
                 >> uart_conf.scndenable
                 >> uart_conf.trainenable
-                >> uart_conf.localenable;
+                >> uart_conf.localenable
```

Comment:
Instead of adding more arguments here and breaking config compatibility, maybe add alternate syntax like `uartfeature main,scnd,radiovolume`? What do you think?

Reply:
Thanks. I didn't noticed backward compatibility issue. I wrongly assumed that `getTokens(N)` will return up to `N` tokens, **and** tokens are counted to the EOL. Now I know how tokens are stored in Parser, and how `getTokens()` works (and how to crash the simulator via ini file ;))

The problem with determining the proper syntax is for the edge case, where the new syntax will define just one feature, ie. `uartfeature main`. But I think I found a quite compatible solution (a0276a3760ae12919ca3595312abf41ad04dee3c)

Please tell me what do you think about my proposal. If you wish I can squash the last commit into 4433ea2e9611b562eb212ddb7104dfc884e004d5 before merging.